pop-cli-for-appchains/guides/running-on-paseo-locally.md

```diff
@@ -417,51 +396,112 @@ curl -H "Content-Type: application/json" \
 http://localhost:8845
 ```
 
-It will take time for your collator to sync with the local Paseo Relay chain.
-
-We now need to onboard the parachain to Paseo.
-
-Go to the Parachains tab.
-
-<figure><img src="../.gitbook/assets/Screenshot 2024-09-24 at 11.37.32 AM.png" alt=""><figcaption><p>Parachains Tab</p></figcaption></figure>
+Ensure the chain is synced with Paseo:
+```bash
+2024-12-10 09:06:05 [Relaychain] Warp sync is complete, continuing with state sync.    
+2024-12-10 09:06:06 [Relaychain] State sync is complete, continuing with block sync.    
+2024-12-10 09:06:06 [Relaychain] 🏆 Imported #47 (0xb04f…12a4 → 0x6adf…6582)    
+2024-12-10 09:06:06 [Relaychain] 🏆 Imported #48 (0x6adf…6582 → 0xd036…e502) 
+```
 
-Select "+ ParaId" and make sure to use the stash account:
+We now need to onboard the chain to Paseo.
 
-<figure><img src="../.gitbook/assets/Screenshot 2024-09-24 at 11.38.09 AM.png" alt=""><figcaption><p>Reserve ParaId</p></figcaption></figure>
+### Reserve Para ID
 
-Next, select "+ ParaThread". Make sure to use the stash account, upload your wasm and genesis state:
+We can reserve a para ID for the chain using pop cli:
 
-<figure><img src="../.gitbook/assets/Screenshot 2024-09-24 at 11.38.50 AM.png" alt=""><figcaption><p>Register ParaThread</p></figcaption></figure>
+```bash
+pop call parachain --url ws://localhost:57731
+```
 
-It will take a moment for the ParaThread to onboard:
+ 
+```bash
+┌   Pop CLI : Call a parachain
+│
+◇  What would you like to do?
+│  Reserve a parachain ID 
+│
+◇  Signer of the extrinsic:
+│  <PUBLIC KEY STASH ACCOUNT>
```

Comment:
That's wrong no?

Reply:
Why?

Reply:
Here is the Secret key URI, not the public. maybe just leave it as `<STASH ACCOUNT>`